backend/wmg/pipeline/summary_cubes/cell_count.py

```diff
@@ -61,6 +66,32 @@ def load(corpus_path: str, df: pd.DataFrame) -> str:
     return uri
 
 
+def return_dataset_dict_w_publications():
+    # cchoi: modifying the dataframe to contain publication coll IDs
```

Comment:
remove this line as its no longer accurate (or update with accurate description)